pom.xml

```diff
@@ -647,6 +647,7 @@
                <configuration>
                  <source>8</source>
                  <doclint>none</doclint>
+                 <includeDependencySources>true</includeDependencySources>
```

Comment:
I think this is what's causing the javadoc error.  The logs are showing issues trying to resolve javadoc on classes from our dependencies.
This is the only change that I've seen (so far) that might cause that.

Reply:
Good catch. I put that in and was experimenting with different options to try to get it to generate links to the Java Servlet API classes, but it didn't work as expected and I forgot to take it out and then I make the other changes on top of it. But that has to be it.

Reply:
I'm going to mark this as resolved. The 1 line deletion that you suggested that I made in commit ID 14fb179 fixed the problem with the workflow.